src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_on_unicast_transports.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTesting.Customization;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.AcceptanceTests.ScenarioDescriptors;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+
+    public class When_publishing_to_scaled_out_subscribers_on_unicast_transports : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Each_event_should_be_delivered_to_single_instance_of_each_subscriber()
+        {
+            await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b => b.When(c => c.SubscribersCounter == 4, async (session, c) =>
```

Comment:
On multicast transports subscribe happens in the FST and is synchronous so in this case I can be sure that when all endpoints are started, subscriptions are there.